Marcs_mod_file/Choosing_a_star.ipynb

```diff
@@ -0,0 +1,128 @@
+{
```

Comment:
<p>Not sure why this text looks different. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/stardis/pull/227/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
fixed in 'typo and format fix'